### ISSUE TITLE
Please unblock these sites

### ISSUE THREAD

**Issue by Ambyjkl** [4skinSkywalker/Anti-Porn-HOSTS-File#73]
I think you misunderstood what I meant in https://github.com/4skinSkywalker/Anti-Porn-HOSTS-File/issues/70. These sites are harmless and should not be in the block list.
fullstory.com
optimizely.com

**Comment by hackerwhale** [comment]
I have removed these sites from HOSTS.txt. I think it will be unblocked now. Please check it.

**Comment by fredowashere** [comment]
I'll look at the PR this evening when I'm back home!
Thanks guys! :D

**Comment by 4skinSkywalker** [comment]
@hackerwhale may I ask you why you removed 25K sites?

**Comment by hackerwhale** [comment]
@4skinSkywalker Sorry for that. I used the vscode editor to open and make changes in the file. But it only shows the option to compare and overwrite which didn't work either. So I switched to GitHub Desktop, cloned the repo, and made the changes again. If there is any issue going on then please let me know. If the sites had been removed then that was not my intention

**Comment by 4skinSkywalker** [comment]
@hackerwhale I still see your PR diff saying "3 additions, 25,525 deletions".
Did you fix?

**Comment by hackerwhale** [comment]
@4skinSkywalker  yes, I have fixed the issue and created a new pull request.

**Comment by 4skinSkywalker** [comment]
@Ambyjkl request addressed in 6288228b0ee8616aabece3b920161d4320dc7f9f
@hackerwhale PR merged in 87e83c4999616749d6a16e16940259cce4029436

**Comment by hackerwhale** [comment]
@4skinSkywalker Thank You
